docker/Dockerfile

```diff
@@ -12,23 +12,57 @@ RUN go get -v github.com/readium/readium-lcp-server/lcpencrypt
 ## Final image
```

Comment:
This comment looks out of date, its not the final image being built here, its the base image. Can you add comments like this for the other images in this dockerfile?

Something like: 
```
###############################################################################
# Base image
###############################################################################
```